loadPluginsToTempDir.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+const _  = require('underscore');
+require('dotenv').config()
+
+const directoryPath = './src/pluginsTemp'
+const separator = require('path').sep
+
+
+const copyFile = (plugin) => {
+  console.log('copying', plugin.path, 'to', plugin.newPath)
+  fs.copyFileSync(plugin.path, plugin.newPath)
+}
+
+const handleError = (error) => {
+  console.log(error)
+}
+
+const hydratePlugin = (plugin) => {
+  var filename = _.last(plugin.path.split(separator))
+
+  plugin.newPath = [directoryPath, filename].join(separator)
+
+  return plugin
+}
+
+const plugins = (environment = process.env) => {
+  if ( !environment.PLUGINS ) { console.log('No plugins are defined.'); return }
+
+  return _.map(JSON.parse(environment.PLUGINS), hydratePlugin)
```

Comment:
Does it make sense to have an environment.PLUGINS.example.js file somewhere to show what this configuration should look like in a way that can be checked into source control?